install/scripts/docker_entrypoint.sh

```diff
@@ -48,7 +48,7 @@ SQL_HOST=127.0.0.1
 SQL_PORT=3306
 
 SERVER_HOST=127.0.0.1
-SERVER_PORT=50051
+SERVER_PORT=${PORT:=50051}
```

Comment:
Can you explain what this syntax is doing?

Reply:
This is setting `SERVER_PORT` to the value of `PORT`.  If `PORT` is unset or null, then it will set to the default value of 50051.

Reply:
Thanks for the explanation!